docs/Developers/Development_Setup.rst

```diff
@@ -91,25 +91,21 @@ https://gitlab.com/coala/coala-utils/
 Installing from Git
 -------------------
 
-We recommend first installing the latest development snapshot of coala's
-master branch from and all of its dependencies with pip3 using
+We recommend installing coala from the master branch for latest updates
+and its dependencies with pip3 using the commands given below. The
+-e tag installs the project in the editable mode from the current path.
```

Comment:
some kind of formatting needed on **-e**

Reply:
How about making it bold?